src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -589,6 +584,18 @@ def report_metrics_for_detectors(
             tags={f"op_{n.allowed_op}": True for n in detected_issues.values()},
         )
 
+    if has_detected_issues:
+        sdk_span.containing_transaction.set_tag("_pi_all_issue_count", len(all_detected_issues))
+        metrics.incr(
+            "performance.performance_issue.aggregate",
+            len(all_detected_issues),
+        )
+        if event_id:
+            sdk_span.containing_transaction.set_tag("_pi_transaction", event_id)
+
+    detected_tags["is_main_project"] = event.project_id in [1]
```

Comment:
I think this is always `False` because detection isn't working for that project, right?

Reply:
We'll find out 😄, in theory we should be putting this into a list that is overridable, but none of this does anything user facing at the moment, so it's probably fine.